autonomy/cli/scaffold_fsm.py

```diff
@@ -418,6 +440,39 @@ def _load_agent_config(self) -> AgentConfig:
         with (Path(self.ctx.cwd) / DEFAULT_AEA_CONFIG_FILE).open() as f:
             return self.ctx.agent_loader.load(f)
 
+    @property
+    def _default_params_config(self) -> Dict:
+        """The default `params` configuration."""
+        abci_app_cls_name = self.dfa.label.split(".")[-1]
+        service_id = abci_app_cls_name.replace(ABCI_APP, "")
+        service_id = re.sub(r"(?<!^)(?=[A-Z])", "_", service_id).lower()
```

Comment:
One day this can all be offered as properties on dfa, with some input validation to ensure these always exist.